packages/serve/package.json

```diff
@@ -14,14 +14,14 @@
     "@webpack-cli/utils": "0.2.2",
     "chalk": "2.4.2",
     "cross-spawn": "6.0.5",
-    "inquirer": "6.5.1",
-    "webpack-dev-server": "^3.8.1"
+    "inquirer": "6.5.1"
   },
   "devDependencies": {
     "@types/cross-spawn": "6.0.0",
     "@types/inquirer": "6.5.0",
     "@types/node": "12.7.2",
-    "typescript": "3.5.3"
+    "typescript": "3.5.3",
+    "webpack-dev-server": "^3.8.2"
   },
   "peerDependencies": {
     "webpack-dev-server": ">=4.0.0"
```

Comment:
No need it here, apparently we missed it somewhere. 

We should put it:
```
"peerDependenciesMeta": {
    "@webpack-cli/serve": {
      "optional": true
    }
  },
```

in webpack-cli main package

Reply:
@evilebottnawi Can we do this in a separate PR? And should it be done with all webpack-cli packages? (e.g. `@webpack-cli/...`)

Reply:
> Can we do this in a separate PR?

Yes, just don't forget

> And should it be done with all webpack-cli packages? 

Maybe, need look in other packages

Reply:
You can have the seperate PRs towards this one @Loonride ?

Reply:
> You can have the seperate PRs towards this one @Loonride ?

Sounds good.

Reply:
Let us know when you've put one up